.github/workflows/release.yml

```diff
@@ -141,7 +141,8 @@ jobs:
              org.cockpit_project.CockpitClient.releases.xml \
              "${{ github.ref_name }}" \
              "$(date +%Y-%m-%d)"
-          git add "$(../src/containers/flatpak/prepare "${DOWNLOAD}" "${CHECKSUM}")"
+          git add "$(../src/containers/flatpak/prepare --packages=upstream --sha256="${CHECKSUM}" "${DOWNLOAD}")"
```

Comment:
Did you really intend "upstream" here? That would make the result somewhat unpredictable. At the time of releasing cockpit it would slurp in the recent releases of podman and machines. I'm okay with trying this for a while, but conceptually it's something that we never tested before. I thought we wanted to bump the releases in a downstream workflow, with a gating test run or so. This would still need to be set up.

So I mostly want to ensure that this is a conscious decision, not just a typo.

Reply:
we definitely need to have a deeper conversation here about what it means to release to flathub.  Also on the agenda for that conversation: what is the version number of the flathub release, and how do we arrange to include points from the changelogs of the other repos (machines, podman, etc.) in the changelog of the flatpak?

Reply:
For now: I imagine that updating other depends can be done manually by pushing more commits to the branch that this creates.  Remember: it just creates the branch — it doesn't PR it.